src/wormhole/_rendezvous.py

```diff
@@ -61,6 +62,54 @@ def buildProtocol(self, addr):
         return proto
 
 
+def mint_hashcash(bits, resource):
+    "1:6:210623:arbitrary string::9WrCxB1SdCBOM3i5:000005"
```

Comment:
What is this line for?

Reply:
Oh, I think that was "note-to-self" about the format .. it should be replaced with a proper docstring for sure :)